grakn-test/src/test/java/ai/grakn/test/graql/analytics/ShortestPathTest.java

```diff
@@ -371,5 +379,6 @@ private void addOntologyAndEntities2() throws GraknValidationException {
 
             graph.commitOnClose();
         }
+        GraknSparkComputer.close();
```

Comment:
This method clears the cached rdd. So I also need it whenever I wanna reload the graph.